aiida/cmdline/commands/cmd_config.py

```diff
@@ -190,3 +194,31 @@ def verdi_config_caching(disabled):
                     echo.echo(identifier)
             elif disabled:
                 echo.echo(identifier)
+
+
+@verdi_config.command('downgrade')
+@click.argument('version', type=int)
```

Comment:
Can we make this a `type=click.Choice` and get the available numbers dynamically from the available migrations? That would make it a lot better from a UI perspective.

Reply:
done